### PR TITLE
Add new palette & text specs

### PR DESCRIPTION
**Pull request by leejiwoo2002**
## Self Checklist

- [x] I wrote a PR title in **English**.
- [x] I added an appropriate **label** to the PR.
- [x] I wrote a commit message in **English**.
- [x] I wrote a commit message according to [**the Conventional Commits specification**](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] [I added the appropriate **changeset**](https://github.com/channel-io/bezier-react/blob/next-v1/CONTRIBUTING.md#add-a-changeset) for the changes.
- [ ] [Component] I wrote **a unit test** about the implementation.
- [ ] [Component] I wrote **a storybook document** about the implementation.
- [ ] [Component] I tested the implementation in **various browsers**.
  - Windows: Chrome, Edge, (Optional) Firefox
  - macOS: Chrome, Edge, Safari, (Optional) Firefox
- [ ] [*New* Component] I added my username to the correct directory in the `CODEOWNERS` file.

## Summary

add new foundation specs for meet feature

## Details

add 2 color palette
- grey50_80
- grey850_80

add 2 text specs
- size30
- size36

add 3 semantic colors
- bg-grey-dim-lightest
- light: grey50_80
- dark: grey850_80

## Breaking change or not (Yes/No)

No

## References

[desk thread](https://desk.channel.io/root/threads/groups/Bezier-62019/6419ace441e99b8c4fa0/6419ace441e99b8c4fa0)
[figma](https://www.figma.com/file/99k33ZxchcPKTz2tzJlZeE/Components?node-id=1711%3A2160&t=CxTXratExzVN1YC9-1)

### CODE REVIEW

**Review by rktguswjd — APPROVED** [review]

File: packages/bezier-react/src/foundation/Colors/Theme/types/SemanticNames.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ type MonoBackgroundColor =
   | `bg-${BaseMonoPaletteKey}-light`
   | `bg-${BaseMonoPaletteKey}-lighter`
   | `bg-${BaseMonoPaletteKey}-lightest`
+  | `bg-${BasePaletteKey.Grey}-dim-lightest`
+  | `bg-${BasePaletteKey.White}-dim-dark`
+  | `bg-${BasePaletteKey.White}-dim-light`
```
Comment (leejiwoo2002) [inline packages/bezier-react/src/foundation/Colors/Theme/types/SemanticNames.ts:13]: BaseMonoPaletteKey를 쓰기엔 아직 커버하지 못하는 색상들이 있어서 우선 별개로 작성해두었습니다

File: .changeset/heavy-rivers-punch.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@channel.io/bezier-react": patch
+---
+
+add new text/palette specs
```
Comment (sungik-choi) [inline .changeset/heavy-rivers-punch.md:5]: 변경사항에 대해 더 자세히 적어주시면 좋을 거 같아요!

**Review by sungik-choi — APPROVED** [review]

File: .changeset/heavy-rivers-punch.md (hunk 2 above)
Comment (leejiwoo2002) [reply .changeset/heavy-rivers-punch.md:5]: PR의 내용을 일부 추가하도록 하겠습니다

Comment (leejiwoo2002) [reply .changeset/heavy-rivers-punch.md:5]: 디테일들 추가했습니다!

**Pull request merged**